addons/stock/models/stock_production_lot.py

```diff
@@ -23,6 +28,8 @@ class ProductionLot(models.Model):
     quant_ids = fields.One2many('stock.quant', 'lot_id', 'Quants', readonly=True)
     product_qty = fields.Float('Quantity', compute='_product_qty')
     note = fields.Html(string='Description')
+    display_complete = fields.Boolean(default=_default_display_complete,
+        compute='_compute_display_complete', store=False)
```

Comment:
let's discuss tomorrow of this tradeoff: a single view, we hide elements in it only if they don't have an id
this could simplify a lot this commit